components/auth/LoginMenu.tsx

```diff
@@ -16,9 +16,14 @@ const LoginMenu = ({ isAuthenticated }: LoginMenuProp) => {
 
   return (
     <>
-      <div className="gc-login-menu pr-3">
```

Comment:
Looks like you might need this ? --- or at least cypress seems to be looking for it

Reply:
Fixing the cypress test to check for element ID's instead of relying on named classes.